<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatMenu</class>
 <widget class="QWidget" name="ChatMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>451</width>
     <height>101</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="textToSend">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>351</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>340</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>19</y>
     <width>441</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>168</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="leaveButton">
      <property name="text">
       <string>Leave</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>textToSend</sender>
   <signal>returnPressed()</signal>
   <receiver>sendButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
